### ISSUE TITLE
Add system property to remove the logo from the log output

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#4019]
Some users may have compelling use cases for needing to remove the jOOQ logo from the log output. They can either do this by muting the `java.util.logging` loggers (or slf4j / log4j, etc), or they should also be able to do this by setting a flag.

The flag is undocumented, so people will need to look for it in the sources.

----
See also:
http://stackoverflow.com/q/28272284/521799

**Comment by ooxi** [comment]
I really like how you handle such things: instead of forcing your users to ugly workarounds you add a simple configuration option and trust your users :+1: 

Such handling drove me to licensing several commercial software packages.

**Comment by lukaseder** [comment]
Thanks, @ooxi, for your nice words.

We did feel some "guilt" when we added that logo, although it is of course perfectly OK for an OSS software to ask for some attribution. If this were UI software, we'd obviously display a splash screen, but with libraries, things are harder. You really don't want a database library to open a splash screen, right? ;)

This particular user (on Stack Overflow) did have a good reason to turn off this logo, and they even replaced the log-logo by a different sort of attribution.

So yes. We don't want to harass our users, at least not too aggressively :)

Looking forward to having you as a commercial customer!
